cli/azd/resources/scaffold/templates/resources.bicept

```diff
@@ -197,11 +197,57 @@ module storageAccount 'br/public:avm/res/storage/storage-account:0.14.3' = {
     ]
     networkAcls: {
       defaultAction: 'Allow'
-    }
+  }
```

Comment:
The white space should not be deleted here. I'll revert this change in my feature branch.